### ISSUE TITLE
Tried to push NAN into the stack when attempting an intercept from a nearly-approaching orbit

### ISSUE THREAD

**Issue by JonnyOThan** [maneatingape/rsvp#6]
current vessel is ion-u [nero].  target is nero (from gpp).

[neeero.zip](https://github.com/maneatingape/rsvp/files/5008920/neeero.zip)

![image](https://user-images.githubusercontent.com/9804078/89074270-f5907f00-d330-11ea-89bd-373717479114.png)

**Comment by maneatingape** [comment]
@JonnyOThan In order to reproduce I'm going to need some more info:
* KSP version
* GPP version
* The version of RSVP that you're using
* The `plan_rsvp` script that is calling the library.

Additionally, your save file is heavily modded and I can't load the relevant craft.
Using the F12 debug menu, can you cheat a *stock* craft into the same orbit (using the "Rendezvous" button). Anything with a kOS processor will do.

Then I can run the same script as you, with a craft in the same location to diagnose the issue.

**Comment by JonnyOThan** [comment]
KSP version: 1.8.1
GPP version: 1.6.4.0
RSVP version: v4

I'm sorry this is so lacking in details.  I had hoped to get another screenshot of the actual orbits but one of the stream viewers had started another burn before I could.  But essentially, imagine a transfer out to something like eeloo or jool but the burn undershot a little - so there is a close approach but no encounter.  In my limited experience with rsvp, running it in this situation could produce a course correction to achieve an encounter, but it resulted in this error instead.

plan_rsvp is:
```
parameter destination is target.
parameter user_options is lexicon().

local options is lexicon(
	"create_maneuver_nodes", (choose "both" if destination:istype("body") else "first"),
	"verbose", true).

if destination:istype("body") {
	set options["final_orbit_periapsis"] to destination:atm:height + 10000.
}

for key in user_options:keys
	set options[key] to user_options[key].

runoncepath("rsvp/main").

print rsvp:goto(destination, options).
```

**Comment by maneatingape** [comment]
@JonnyOThan Thanks for the details, I was able to recreate the issue.

Steps:
* KSP 1.8.1, GPP 1.6.4, kOS 1.2.1, RSVP v4
* Fresh Sandbox game
* Launched a test craft `Issue #6` into  orbit
* Edited `FLIGHTSTATE.UT` to `5524319701.597928` to match savefile.
* Edited `VESSEL.ORBIT` parameters to match `ION-U [Nero]` craft from savefile.

This looks like your description of the situation:
![screenshot0](https://user-images.githubusercontent.com/44142177/89121060-8db67180-d4b3-11ea-8dfd-370af4f617db.png)

The expected behavior is that you should just be able to run the script again to refine the intercept.
Will investigate the `NaN` error.

**Comment by maneatingape** [comment]
Fixed in c0c59dd1ed52bcb6d3db8ec8a37799350243db9e and [Release 5](https://github.com/maneatingape/rsvp/releases/tag/v5)

Running a new time-limited search:
```
local options is lex(
    "search_duration", 400 * 21600,
    "create_maneuver_nodes", "both",
    "verbose", true
).

print rsvp:goto(nero, options).
```

Resulted in a decent correction and intercept:
![screenshot1](https://user-images.githubusercontent.com/44142177/89314216-44138580-d671-11ea-8466-c2c3b89a0f52.png)

**Comment by maneatingape** [comment]
@JonnyOThan Appreciate all the feedback btw, keep it coming!

**Comment by JonnyOThan** [comment]
Hm, the code is slightly different but I'm still hitting a similar error in the same situation.  Vessel is "Leto Relay"
[rsvp_bug.zip](https://github.com/maneatingape/rsvp/files/5065522/rsvp_bug.zip)

**Comment by maneatingape** [comment]
This is due to a bug, caused by using `min` instead of `max`in the Lambert guard condition.
Should be fixed in 4067861bf78a128a3a3dc2dcd7e0bb80fbd6f258.

Would you mind trying again using the latest master branch?
